components/cam/bld/namelist_files/namelist_definition.xml

```diff
@@ -4204,12 +4204,6 @@ with modal chemistry.
 Default: none
 </entry>
 
-<entry id="oldcldoptics" type="logical"  category="radiation"
```

Comment:
I like the idea of removing this, but is this gonna be a problem for anything on the E3SM side? If not then we might want to just make a parallel PR there to get rid of this option.

Reply:
It's not used in any of our testing compsets as far as I can tell (according to `grep`), so for all practical purposes it doesn't exist anyways. I'll make another PR upstream.